auditlogger.go

```diff
@@ -0,0 +1,58 @@
+package ladon
+
+import (
+	"log"
+	"os"
+	"strings"
+)
+
+// AuditLogger tracks denied and granted authorizations.
+type AuditLogger interface {
+	LogRejectedAccessRequest(request *Request, pool Policies, deciders Policies)
+	LogGrantedAccessRequest(request *Request, pool Policies, deciders Policies)
+}
+
+// AuditLoggerNoOp is the default AuditLogger, that tracks nothing.
+type AuditLoggerNoOp struct{}
+
+func (*AuditLoggerNoOp) LogRejectedAccessRequest(r *Request, p Policies, d Policies) {}
+func (*AuditLoggerNoOp) LogGrantedAccessRequest(r *Request, p Policies, d Policies)  {}
+
+var DefaultAuditLogger = &AuditLoggerNoOp{}
+
+// AuditLoggerInfo outputs information about granting or rejecting policies.
+type AuditLoggerInfo struct {
```

Comment:
How about `AuditLoggerDefault` or `DefaultAuditLogger`?

Reply:
Also, just a technicality, but could you move this to `audit_logger_default`?

Reply:
I thought the NoOp would be the default one

Reply:
Right, that makes sense, let's keep it as is!